AppCreationScripts/apps.json

```diff
@@ -36,11 +36,12 @@
         "settingFile": "/JavaScriptSPA/config.js", 
             "replaceTokens": 
               {
-            "clientId": "Enter_the_Application_Id_Here",
-            "authority": "Enter_the_Cloud_Instance_Here/Enter_the_Tenant_Info_Here",
-            "redirectUri": "Enter_the_Redirect_Uri_Here",
-            "graphMeEndpoint": "Enter_the_Graph_Endpoint_Here/v1.0/me",
-            "graphMailEndpoint": "Enter_the_Graph_Endpoint_Here/v1.0/me/messages"
+              "appId": "Enter_the_Application_Id_Here",
+              "tenantId": "Enter_the_Tenant_Info_Here",
+              "authorityEndpointHost": "Enter_the_Cloud_Instance_Id_Here",
+              "redirectUri": "Enter_the_Redirect_Uri_Here",
+              "msgraphEndpointHost": "Enter_the_Graph_Endpoint_Here/v1.0/me",
```

Comment:
Can we have two separate objects for configuration for token acquisition (msal js) and token usage(graph endpoint here)? msal js is designed to take a config object for initialization and appending resource end points to this confuses users. This is the same problem we are trying to solve for B2C samples too.

Reply:
I see -good idea.

Reply:
applied the changes.